example/src/main/resources/microsite/data/menu.yml

```diff
@@ -52,3 +52,6 @@ options:
     - title: These
       url: data/These.html
       menu_section: data
+    - title: Kleisli
+      url: data/Kleisli.html
+      menu_section: scalaz
```

Comment:
`menu_section: data`, please :)

Also, keep this list sorted alphabetically by title, please :)

Reply:
I'll address this in the next PR - great catch